plugins/embark-assistant/help_ui.cpp

```diff
@@ -250,13 +253,18 @@ namespace embark_assist{
                 help_text.push_back("  Flux determination is made by finding the reaction PIG_IRON_MAKING.");
                 help_text.push_back("- Right world map overlay not implemented as author has failed to");
                 help_text.push_back("  emulate the sizing logic exactly.");
-                help_text.push_back("Version 0.1 2017-08-30");
+                help_text.push_back("- There's currently a DF bug that causes adamantine spires reaching");
+                help_text.push_back("  that have been removed at world gen to fail to be generated. It's likely");
+                help_text.push_back("  this bug also affects magma pools. This plugin does not address this.");
+                help_text.push_back("Version 0.3 2018-02-26");
 
                 break;
             }
 
             //  Add control keys to first line.
-            embark_assist::screen::paintString(pen_lr, 1, 1, "TAB/Shift-TAB");
+            embark_assist::screen::paintString(pen_lr, 1, 1, "TAB");
```

Comment:
I didn't notice this before, but you should be passing interface_key enum items into Screen::getKeyDisplay somehow instead of hardcoding the human-readable form here. There are examples of OutputString and related functions in plugins/uicommon.h. Doing that would avoid confusion if people have changed their keybindings, and would also be more consistent (DF uses `Tab`, `Enter`, `Esc`, etc.).

Reply:
I would have if I'd known how to get at it. I do in Lua scripts. I'll look at the reference.